examples/general/abs_home_path/src/lib.rs

```diff
@@ -6,7 +6,7 @@ extern crate rustc_ast;
 extern crate rustc_hir;
 extern crate rustc_span;
 
-use clippy_utils::{diagnostics::span_lint, match_def_path, path_def_id};
+use clippy_utils::{diagnostics::span_lint, match_def_path, path_def_id, source::HasSession};
```

Comment:
This is for the `sess` method?

Could we please use [`LintContext`](https://doc.rust-lang.org/beta/nightly-rustc/rustc_lint/trait.LintContext.html) instead?

I think `LintContext` is more widely used and is less likely to change.